tests/robustness/multiclient_test/multiclient_test.go

```diff
@@ -205,6 +205,17 @@ func TestMaintenanceAction(t *testing.T) {
 	require.NoError(t, err)
 }
 
+func TestDeleteRandomSnapshotAction(t *testing.T) {
+	const numClients = 4
```

Comment:
Does this delete 4 snapshots at once in the common case? Isn't that too agressive?

Reply:
yes, reduced to 1.